src/logic/documents/scorecards.js

```diff
@@ -448,6 +460,100 @@ const scorecard = ({
   ];
 };
 
+const coverSheet = ({
+  competitionName,
+  activityCode,
+  numberOfScorecards,
+  room,
+}) => {
+  const { eventId, roundNumber, groupNumber } = activityCode
+    ? parseActivityCode(activityCode)
+    : {};
+
+  return [
+    {
+      text: competitionName,
+      bold: true,
+      fontSize: 15,
+      margin: [0, 6],
+      alignment: 'center',
+    },
+    {
+      text: `${eventNameById(eventId)} Round ${roundNumber}`,
+      fontSize: 15,
+      margin: [0, 6],
+      alignment: 'center',
+    },
+    {
+      text: `${room.name}: Group ${groupNumber}`, // TODO: make aware of room/stage
+      fontSize: 15,
+      margin: [0, 6],
+      alignment: 'center',
+    },
+    {
+      text: '-------------------- FOR DELEGATE --------------------',
+      alignment: 'center',
+      margin: [0, 6],
+    },
+    {
+      text: [
+        '1. Bundled all ',
+        { text: numberOfScorecards, bold: true },
+        ' scorecards ',
+        { text: '□', font: 'WenQuanYiZenHei' },
+      ],
+      fontSize: 10,
+      alignment: 'left',
+      margin: [20, 6, 0, 6],
+    },
+    {
+      text: [
+        '2. Checked for missing signatures ',
+        { text: '□', font: 'WenQuanYiZenHei' },
+      ],
+      fontSize: 10,
+      font: 'WenQuanYiZenHei',
+      alignment: 'left',
+      margin: [20, 6, 0, 6],
+    },
+    {
+      text: '3. Number of scorecards with incidents: ______',
+      fontSize: 10,
+      margin: [20, 6, 0, 6],
+    },
+    initialsField(),
+    {
+      text: '-------------------- FOR DATA ENTRY --------------------',
+      alignment: 'center',
+      margin: [0, 6],
+    },
+    {
+      text: '4. Results entered by Scoretaker',
+      fontSize: 10,
+      margin: [20, 6, 0, 6],
+    },
+    initialsField('Scoretaker'),
+    {
+      text: '5. Incidents logged by Delegate',
+      fontSize: 10,
+      margin: [20, 6, 0, 6],
+    },
+    initialsField(),
+    {
+      text: '6. Results checked by Delegate',
+      fontSize: 10,
+      margin: [20, 6, 0, 6],
+    },
+    initialsField(),
+  ];
+};
+
+const initialsField = (person = 'Delegate') => ({
```

Comment:
I would pass this explicitly :)

```suggestion
const initialsField = (person) => ({
```

Reply:
Will do!